release-team/README.md

```diff
@@ -141,41 +141,11 @@ If you're interested in learning more about how the Release Team is selected, as
 
 ## Milestone Maintainers
 
-Across release cycles, one of the best signals for [issue triage](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#milestones) is whether or not an issue or PR is actually targeted for the current milestone.
+The [milestone maintainers](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md#milestone-maintainers) are responsible to add, adapt or remove a milestone, e.g. the next major Kubernetes release, at an enhancement, issue, PR.
```

Comment:
"are responsible to add" ==> "are responsible for adding, adapting, or removing a milestone"

"at an enhancement" ==> **on** an enhancement